l10n_nl_xaf_auditfile_export/models/xaf_auditfile_export.py

```diff
@@ -43,7 +43,7 @@ def _compute_auditfile_name(self):
             item.auditfile_name = '%s.xaf' % item.name
             if item.auditfile:
                 auditfile = base64.b64decode(item.auditfile)
-                zf = StringIO(auditfile)
+                zf = BytesIO(auditfile)
```

Comment:
This is now in line with [10.0](https://github.com/OCA/l10n-netherlands/blob/10.0/l10n_nl_xaf_auditfile_export/tests/test_l10n_nl_xaf_auditfile_export.py#L42)